src/pybind/mgr/dashboard/controllers/prometheus.py

```diff
@@ -61,20 +61,22 @@ def get_access_info(self, module_name):
         user = None
         password = None
         cert_file = None
-        secure_monitoring_stack = bool(mgr.get_module_option_ex('cephadm',
-                                                                'secure_monitoring_stack',
-                                                                'false'))
-        if secure_monitoring_stack:
-            cmd = {'prefix': f'orch {module_name} get-credentials'}
-            ret, out, _ = mgr.mon_command(cmd)
-            if ret == 0 and out is not None:
-                access_info = json.loads(out)
-                user = access_info['user']
-                password = access_info['password']
-                certificate = access_info['certificate']
-                cert_file = tempfile.NamedTemporaryFile(delete=False)
-                cert_file.write(certificate.encode('utf-8'))
-                cert_file.flush()
+
+        orch_backend = mgr.get_module_option_ex('orchestrator', 'orchestrator')
```

Comment:
```
FAILED tests/test_prometheus.py::PrometheusControllerTest::test_add_silence
FAILED tests/test_prometheus.py::PrometheusControllerTest::test_expire_silence
FAILED tests/test_prometheus.py::PrometheusControllerTest::test_get_silences
FAILED tests/test_prometheus.py::PrometheusControllerTest::test_list - Assert...
FAILED tests/test_prometheus.py::PrometheusControllerTest::test_rules - Asser...
FAILED tests/test_prometheus.py::PrometheusControllerTest::test_update_silence
```
a bunch of tests failed off of this. I didn't look closely but I think they might be mocking this `get_module_option_ex` and expect it to get 3 arguments and be called once with the args for the secure monitoring stack check. So the tests will need to be updated with this change.

Reply:
@adk3798  was updating the UT... I added some more test cases to force both paths: cephadm and rook